tests/testthat/test-nest.R

```diff
@@ -315,6 +315,18 @@ test_that("warn about old style interface", {
   expect_named(out, c("x", "data"))
 })
 
+test_that("only warn about unnamed inputs (#1175)", {
+  df <- tibble(x = 1:3, y = 1:3, z = 1:3)
+  expect_warning(out <- nest(df, x, y, foo = z), "data = c(x, y)", fixed = TRUE)
```

Comment:
Would these be better as snapshot tests?

Reply:
Yes, but there are quite a few in this file so I will do a separate PR and move them all over to `expect_snapshot()`